src/Test/Utilities/Desktop/ThrowingTraceListener.cs

```diff
@@ -0,0 +1,36 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System;
+using System.Diagnostics;
+using Xunit;
+
+namespace Microsoft.CodeAnalysis
+{
+    public sealed class ThrowingTraceListener : System.Diagnostics.TraceListener
```

Comment:
We could; I'd be terrified that somebody will inherit from 'TraceListener' and then inherit from this by accident.